docs/presto-admin-commands.rst

```diff
@@ -489,14 +489,14 @@ server upgrade
 **************
 ::
 
-    presto-admin server upgrade local_package_path [local_config_dir]
+    presto-admin server upgrade new-rpm [local_config_dir]
```

Comment:
I second maciej's suggestion of path/to/new/package.rpm. It makes it clear that it it's a path to an rpm file.